test/integration/ratelimit_test.go

```diff
@@ -52,7 +53,7 @@ func TestDuplicateFQDNRateLimit(t *testing.T) {
 			PasswordFile: "test/secrets/ratelimits_redis_password",
 		}
 
-		fc := clock.NewFake()
+		fc := clock.New()
```

Comment:
Why does this need to be a real clock now?

Reply:
Because the integration test stack uses system time, not a fake clock that we have control over. Now if you're like, "hey doesn't that mean that this test was flawed to begin with?" You're right, this integration test never actually worked. We just thought it did because, based on now being the unix epoch, it agreed that there was no room in the TAT for another request. Trouble is, it would have said the same thing had we checked before issuing the second certificate.

Reply:
Well that's exciting. Good catch!

Reply:
Thanks!